test/apify/enqueue_links/enqueue_links.test.ts

```diff
@@ -150,14 +161,22 @@ describe('enqueueLinks()', () => {
             };
 
             const pseudoUrls = [
-                { purl: 'https://example.com/[(\\w|-|/)*]', method: 'POST' as const },
-                { purl: '[http|https]://cool.com/', userData: { foo: 'bar' } },
+                { purl: 'https://example.com/[(\\w|-|/)*]' },
+                { purl: '[http|https]://cool.com/' },
             ];
 
             await browserCrawlerEnqueueLinks({
                 options: { selector: '.click', pseudoUrls },
                 page,
                 requestQueue,
+                transformRequestFunction: (request) => {
+                    if (/example\.com/.test(request.url)) {
+                        request.method = 'POST';
+                    } else if (/cool\.com/.test(request.url)) {
+                        request.userData.foo = 'bar';
+                    }
+                    return request;
+                },
```

Comment:
btw i would still keep a test like this too, so we know it works both ways

Reply:
yup, will do both